src/test/java/de/tum/in/www1/artemis/domain/FeedbackArchitectureTest.java

```diff
@@ -0,0 +1,29 @@
+package de.tum.in.www1.artemis.domain;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.noClasses;
+
+import org.junit.jupiter.api.Test;
+
+import com.tngtech.archunit.lang.ArchRule;
+
+import de.tum.in.www1.artemis.AbstractArchitectureTest;
+import de.tum.in.www1.artemis.service.FeedbackService;
+
+class FeedbackArchitectureTest extends AbstractArchitectureTest {
+
+    @Test
+    void testSetLongFeedbackTextNotUsed() {
+        ArchRule toListUsage = noClasses().should().callMethod(Feedback.class, "setLongFeedbackText").because("this method should only be used by JPA. Use setDetailText instead.");
+        toListUsage.check(allClasses);
+        toListUsage.check(testClasses);
```

Comment:
allClasses includes testClasses. Also the variable name does not match the rule.
```suggestion
        ArchRule setLongFeedbackText = noClasses().should().callMethod(Feedback.class, "setLongFeedbackText").because("this method should only be used by JPA. Use setDetailText instead.");
        setLongFeedbackText.check(allClasses);
```